tests/test_data_processor.py

```
@@ -0,0 +1,42 @@
+import pytest
+import pandas as pd
+import numpy as np
+from src.diabetes.data_processor import DataProcessor
+
+
+def test_load_data(data_processor, sample_data):
+    assert data_processor.df.equals(sample_data)
+
+def test_preprocess_data(data_processor):
+    data_processor.preprocess_data()
+    
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)